dns/UnboundBL/src/opnsense/mvc/app/controllers/OPNsense/UnboundBL/Api/ServiceController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace OPNsense\UnboundBL\Api;
```

Comment:
I'm not sure if UnboundBL will lead into trouble, if it works for you, ok, but it should be Unboundbl .. probably wait for @fichtner before you change everything again.